src/utils/PermissionUtils.js

```diff
@@ -2,8 +2,8 @@ module.exports = class PermissionUtils {
   static isDeveloper (client, user) {
     const botGuild = client.guilds.get(process.env.BOT_GUILD)
     const developerRole = botGuild && botGuild.roles.get(process.env.DEVELOPER_ROLE)
-    const hasRole = developerRole && developerRole.members.has(user.id)
-    return hasRole
+    const isDeveloper = (developerRole && developerRole.members.has(user.id)) || (user.id === process.env.DEVELOPER_USER)
```

Comment:
what if I'd like to set multiple users as developers, but not on a whole role from a guild?

Reply:
Done.